translator/translate/metrics/global_dimensions/ruleGlobalDimensions.go

```diff
@@ -0,0 +1,41 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package global_dimensions
+
+import (
+	parent "github.com/aws/amazon-cloudwatch-agent/translator/translate/metrics"
+)
+
+type globalDimensions struct {
+}
+
+const SectionKey = "global_dimensions"
+
+func (ad *globalDimensions) ApplyRule(input interface{}) (returnKey string, returnVal interface{}) {
+	im := input.(map[string]interface{})
+
+	dimensions := map[string]interface{}{}
+
+	if _, ok := im[SectionKey]; !ok {
+		returnKey = ""
+		returnVal = ""
+	} else {
```

Comment:
I _think_ that this isn't necessary, but I am a little fuzzy on the syntax here because I don't like named returns in Go. 

My understanding is you could simplify this to:
```go
if _, ok := im[SectionKey]; !ok {
    return
}

// rest of the logic, no else
```

Reply:
Again, this is the pattern used all over the place in the surrounding files which is why I've used it. If you'd like me to change it then I'm happy to, but it might look a bit strange compared to any of the other files in the `/translate` part of the file tree.

Reply:
Having just tested this, there is a subtle difference between assigning to empty strings vs just calling `return` in that the values are `nil` if you don't assign them. I've stuck with `""` in case that's important for some reason later on in the processing.

Reply:
Sorry I keep pushing for things that are different from what's around it. I appreciate that you looked into it for me. Resolving